File: files/200-299/227/CH-227.py
import pandas as pd

path = "CH-227 Data Normalization Min-Max.xlsx"

input = pd.read_excel(path, usecols="B:G", skiprows=1, nrows=7)
test = pd.read_excel(path, usecols="J:O", skiprows=1, nrows=7).rename(columns=lambda col: col.split('.1')[0] if '.1' in col else col)

result = (input - input.min()) / (input.max() - input.min())

print(result.equals(test)) # True

File: files/200-299/227/CH-227.R
library(tidyverse)
library(readxl)

path = "files/CH-227 Data Normalization Min-Max.xlsx"
input = read_excel(path, range = "B2:G8")
test = read_excel(path, range = "J2:O8")

result = input %>%
  mutate(across(where(is.numeric), ~ (. - min(.)) / (max(.) - min(.))))

all.equal(result, test)
#> [1] TRUE
